python/paddle/v2/fluid/io.py

```diff
@@ -283,25 +361,13 @@ def save_inference_model(dirname,
     with open(model_file_name, "wb") as f:
         f.write(inference_program.desc.serialize_to_string())
 
-    save_params(executor, dirname, main_program)
-
-
-def load_persistables_if_exist(executor, dirname, main_program=None):
-    filenames = next(os.walk(dirname))[2]
-    filenames = set(filenames)
-
-    def _is_presistable_and_exist_(var):
-        if not is_persistable(var):
-            return False
-        else:
-            return var.name in filenames
-
-    load_vars(
+    parameter_list = get_parameters(inference_program)
+    save_vars(
```

Comment:
We can try to call `save_persistables` here.

Reply:
Done